tests/infra/path.py

```diff
@@ -86,6 +87,14 @@ def copy_dir(src_path, dst_path):
     copy2(src_path, dst_path)
 
 
+def compute_file_checksum(file_name):
```

Comment:
fwiw, https://pypi.org/project/filehash/ can do a directory at once with a pattern filter.

Reply:
I haven't used this just yet but I'm happy to looking into this further and improve the verification logic in a separate issue!